hw5/hw5.py

```diff
@@ -0,0 +1,227 @@
+
+
+import pandas as pd
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+import math
+import matplotlib.font_manager as font_manager
+import matplotlib.ticker as ticker
+from matplotlib.patches import ConnectionPatch
+
+
+def read_bed6(path, headers = ["chromosome","start", "end", "name", "score", "strand"]):
+    alignment = pd.read_table(path, header=None, names = headers)
+    return alignment
+
+alignment = read_bed6("/content/alignment.bed")
```

Comment:
Это, видимо, осталось после использования коллаба? Но вообще оставлять абсолютные пути в своём коде это не очень хорошо. В идеале, если ты выкладываешь код в репозиторий, то он должен быть самодостаточным. В репозитории должны лежать данные с которыми код работает, и код тоже должен брать эти данные "из коробки", без дополнительных изменений
```suggestion
alignment = read_bed6("content/alignment.bed")
```